astroquery/mast/tests/test_mast_remote.py

```diff
@@ -964,6 +964,11 @@ def test_tesscut_get_cutouts(self, product):
         assert len(cutout_hdus_list) >= 1
         assert isinstance(cutout_hdus_list[0], fits.HDUList)
 
+        # Check that an INFO message is returned when timeout is adjusted
+        mast.Tesscut.get_cutouts(product=product, coordinates=coord, size=5, timeout=1000)
```

Comment:
Maybe assert that the `TIMEOUT` attribute has been properly set back to the default one.